tests/marker_test.go

```diff
@@ -68,15 +48,13 @@ var _ = Describe("bridge-marker", func() {
 				return true
 			}, 20*time.Second, 5*time.Second).Should(Equal(true))
 
-			out, err = tests.RunOnNode(node, fmt.Sprintf("sudo ip link del %s", uniqueBridgeName))
-			if err != nil {
-				panic(fmt.Errorf("%v: %s", err, out))
-			}
+			err =tests.RemoveBridgeFromNode(node,uniqueBridgeName)
```

Comment:
missing space after equals ("=tests")

Reply:
done I update the make go run fmt and vet on the test and cmd folder as well